iothub/service/src/Messaging/TransportType.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Transport types supported by MessagingClient.
```

Comment:
This should be used by FileUploadNotificationProcessor and MessageFeedbackProcessor as well